lib/src/components/ComboBox/Select.tsx

```diff
@@ -0,0 +1,262 @@
+import { assignInlineVars } from "@vanilla-extract/dynamic";
+import {
+  ChangeEvent,
+  FC,
+  forwardRef,
+  ForwardRefRenderFunction,
+  KeyboardEvent,
+  MouseEvent,
+  MutableRefObject,
+  useEffect,
+  useRef,
+  useState,
+} from "react";
+import { useClickOutside } from "src/hooks/useClickOutside";
+import { Flex } from "../Flex";
+import {
+  selectContainerStyles,
+  selectErrorMsg,
+  selectIconRecipe,
+  selectInputRecipe,
+  selectListContainerStyle,
+  selectListRecipe,
+  selectVars,
+  inputTextContainer,
+  noDataFoundStyles,
+  selectPlaceholderRecipe,
+  inputRecipe,
+} from "./select.css";
+import { SelectPropsType, OptionsType } from "./select.types";
+import { ArrowDown, Clear } from "./components/SelectIcons";
+
+const SelectComponent: ForwardRefRenderFunction<
+  HTMLDivElement,
+  SelectPropsType
+> = (
+  {
+    placeholder = "Pick one",
```

Comment:
Do not keep default placeholders, it results un-intentional behaviour keep it blank

Reply:
ok